pennylane/templates/layers/custom_entangler.py

```diff
@@ -249,4 +249,4 @@ def circuit(weights):
         if coupling_weights is not None:
             broadcast(unitary=coupling, pattern=pattern, wires=wires, parameters=coupling_weights[layer])
         else:
-            broadcast(unitary=coupling, pattern=pattern, wires=wires)
\ No newline at end of file
+            broadcast(unitary=coupling, pattern=pattern, wires=wires)
```

Comment:
```suggestion
            broadcast(unitary=coupling, pattern=pattern, wires=wires)
            
```

Python standard to add a blank line :)